Gemfile

```diff
@@ -163,4 +167,8 @@ group :development, :test do
   gem 'stub_env', '~> 1.0.4'
   gem 'webdrivers', '~> 5'
   gem 'webmock', '~> 3.8'
+
+  # s3 Dependencies
+  # gem 'browse-everything'
+  gem 'minio-ruby'
```

Comment:
i don't think we need this? the aws-sdk should interact with minio out of the box